core/app/[locale]/(default)/cart/_components/shipping-info/fragment.ts

```diff
@@ -1,27 +1,34 @@
 import { graphql } from '~/client/graphql';
 
-export const ShippingInfoFragment = graphql(`
-  fragment ShippingInfoFragment on Checkout {
-    entityId
-    shippingConsignments {
+import { ShippingOptionsFragment } from '../shipping-options/fragment';
+
+export const ShippingInfoFragment = graphql(
+  `
+    fragment ShippingInfoFragment on Checkout {
       entityId
-      selectedShippingOption {
+      shippingConsignments {
```

Comment:
Should we update this in a different PR? 🥷

Reply:
I'd consider this like fixing eslint issues as this was caught in a new version bump.